### PR TITLE
trying to pull search form fields from FHIR resource

### PR DESCRIPTION
**Pull request by sovello**

### CODE REVIEW

File: ui/src/views/People/Search.vue [hunk 1]
```diff
@@ -87,24 +84,83 @@ export default {
...
+      //     id: "expression",
+      //     max: 1,
+      //     name: "expression",
+      //     required: false,
+      //     type: "string",
+      //     value: null
+      //   }
+      // ],
+      fields: [],
+      name: { default: "Mkulima"},
```
Comment (sovello) [inline ui/src/views/People/Search.vue]: ignore this name object

**Comment by sovello** [comment]
Yah, the iHRIS prefix is done,
You can see the form now, but there's an issue with rendering Reference Fields. Am yet to look into it, but something inside components.Form.Reference.created

**Comment by lokisapocalypse** [comment]
Code looks good. I did find a bug however and something that needs to be updated.

1) If there are no search results matched, it tries to run forEach on something that doesn't exist.

2) Can you populate the data in the table? When I did a search by surname, only surname showed up in the search results.

**Comment by sovello** [comment]
yes. lots of calls have to be made. let me take a look at that.

**Pull request merged**